aws-opentelemetry-distro/tests/amazon/opentelemetry/distro/test_aws_metric_attribute_generator.py

```diff
@@ -773,6 +774,45 @@ def test_both_metric_when_local_root_consumer_process(self):
         self.assertIsNotNone(service_attributes)
         self.assertIsNotNone(dependency_attributes)
 
+    def test_db_user_attribute(self):
```

Comment:
Please match tests to Java tests, once updated.

Reply:
Will update